community/unsafe/src/main/java/org/neo4j/unsafe/impl/internal/dragons/MemoryManager.java

```diff
@@ -32,8 +32,7 @@
     /**
      * The amount of memory, in bytes, to grab in each Slab.
      */
-    private static final long GRAB_SIZE = Integer.getInteger(
-            MemoryManager.class.getName() + ".GRAB_SIZE", 512 * 1024 ); // 512 KiB
+    private static final long GRAB_SIZE = FeatureToggles.getInteger( "GRAB_SIZE", 512 * 1024 ); // 512 KiB
```

Comment:
That might be a fair thing to avoid. But I would still like you to motivate why you think it should be avoided.